FluidNC/src/Configuration/Tokenizer.cpp

```diff
@@ -82,6 +82,17 @@ namespace Configuration {
                 goto parseAgain;
 
             default:
+                if (IsListItem()) {
+                  token_.is_list_ = true;
+                  Inc();
+                  // skip expected whitespace. TODO: validate that it's indeed a whitespace
```

Comment:
Reason: It could also be a plain scalar (e.g. [`-42`](https://yaml.org/spec/1.2.2/#block-sequences))

Reply:
But did this YAML parser support that before? I guess not, it always relied on "key: value" pairs.